pdf/model/font.go

```diff
@@ -71,15 +71,48 @@ func DefaultFont() *PdfFont {
 }
 
 // NewStandard14Font returns the standard 14 font named `basefont` as a *PdfFont, or an error if it
-// `basefont` is not one the standard 14 font names.
+// `basefont` is not one of the standard 14 font names.
 func NewStandard14Font(basefont string) (*PdfFont, error) {
```

Comment:
How about making basefont Standard14Font here as well?
And scrap the NewStandard14FontMustCompile.
Just log an ERROR (common.Log.Error if not existent)...
only way to go wrong here is like
NewStandard14Font(Standard14Font("does not exist"))
which seems unlikely to happen without a motive.